extensions/amp-layout/validator-amp-layout.protoascii

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-layout
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-layout"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-layout>
+  html_format: AMP
+  tag_name: "AMP-LAYOUT"
+  requires_extension: "amp-layout"
+  attr_lists: "extended-amp-global"
+  attr_lists: "mandatory-src-or-srcset"
```

Comment:
Oh whoops, that line lingered from when I fixed a merge conflict, my bad - I'll update my branch right away